ui/package.json

```diff
@@ -15,7 +15,8 @@
     "benchmark": "NODE_ENV=production BROWSERSLIST='Chrome > 100' GENERATE_SOURCEMAP=false ts-node --experimental-specifier-resolution=node scripts/run-benchmark.mts",
     "prettier": "prettier --write --ignore-path ../.prettierignore 'packages/**/*.{js,jsx,ts,tsx,md,mdx}'",
     "storybook": "storybook dev -p 6006",
-    "build-storybook": "storybook build"
+    "build-storybook": "storybook build",
+    "chromatic": "chromatic --project-token 19693a33e466"
```

Comment:
Monica, can you remove the project token from here and pass it through env?

I guess, It would be even better if we can deprecate this token and delete it, as it has got into git already.

Reply:
I reset the token on Chromatic so everything should be ok now